gui/elm/Window/Tray/Dashboard.elm

```diff
@@ -296,3 +299,53 @@ removeCurrentAction model =
 currentUserAction : Model -> Maybe UserAction
 currentUserAction model =
     List.head model.userActions
+
+
+viewActionContentLine : String -> List (Html Msg)
+viewActionContentLine line =
+    let
+        decorated =
+            Regex.find decorationRegex line
+                |> List.map decoratedName
+
+        rest =
+            Regex.split decorationRegex line
+                |> List.map text
+    in
+    List.intersperseList decorated rest
+
+
+decorationRegex : Regex.Regex
+decorationRegex =
+    -- Find all groups of characters between backticks
+    Maybe.withDefault Regex.never <|
+        Regex.fromString "`(.+?)`"
+
+
+decoratedName : Regex.Match -> Html Msg
+decoratedName match =
+    match
+        |> .submatches
+        |> List.head
+        |> Maybe.withDefault (Just "")
+        |> Maybe.map viewName
+        |> Maybe.withDefault (text "")
```

Comment:
Do we really need two `Maybe.withDefault`? I think we can remove the first one

Reply:
We need both:
1. `.submatches` returns a `List (Maybe String)`
2. so `List.head` returns a `Maybe (Maybe String)`

The first one could be removed if `viewName` would take a `Maybe String` rather than a `String` as parameter but I'd prefer to avoid dealing with maybes in this function.

Reply:
I rewrote it with a switch case as I can pattern match `Just (Just str)` instead of using `Maybe.withDefault` twice.